forms-flow-api/src/formsflow_api/resources/application.py

```diff
@@ -301,10 +301,7 @@ class ApplicationResourcesByIds(Resource):
     @staticmethod
```

Comment:
Looks like no changes related to this PR on this file, except some format changes which is highly dependant on the IDE. Please revert this.